### PR TITLE
SPCTR-2394: Fix image responsive sizing issue for mobile devices

### PR DESCRIPTION
**Pull request by chandan-30**
### Description
This PR addresses the issue by fixing a problem with image sizing on mobile devices. Currently, even with responsive sizing set up for images, browsers are still downloading the large size image on mobile, causing slower load times and increased data usage.

To address this issue, this PR introduces changes to the code that allow for different image sizes to be served based on the user's device and viewport size. This helps to ensure that the optimal image size is being downloaded on mobile devices, without compromising on image quality or user experience.

### Screenshots
https://d.pr/v/LP8G6G

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/image/save.js [hunk 1]
```diff
@@ -36,6 +36,7 @@ export default function Save( props ) {
 			srcSet={ `${ url } ${ urlTablet ? ',' + urlTablet + ' 780w' : '' }${
 				urlMobile ? ', ' + urlMobile + ' 360w' : ''
 			}` }
+			sizes="(max-width: 480px) 150px"
```
Comment (AbhijitNage123) [inline src/blocks/image/save.js:39]: We are making changes in save.js. will need to add deprecated file for image. Also make sure you have tested backward case i. e creating image with diff images sizes option on responsive devices in master and the switch to your branch. And verify validation issue occurred or not. if occurred then add deprecated file.

Comment (chandan-30) [reply src/blocks/image/save.js:39]: Added deprecated files and I tested it for backward case, got no validation errors. Please review.

**Review by AbhijitNage123 — APPROVED** [review]

**Comment by supriyajjadhav** [comment]
@chandan-30 As I can see there are no changes in attributes.js So we can use the existing attributes file instead of creating new. Please remove attributes.js file

**Comment by chandan-30** [comment]
@supriyajjadhav While there haven't been any modifications to the attributes.js file located in the deprecated folder, we decided to include it in order to maintain consistency.

**Comment by RajkiranBagal** [comment]
> @supriyajjadhav While there haven't been any modifications to the attributes.js file located in the deprecated folder, we decided to include it in order to maintain consistency.

We can remove it, if not needed

**Review by RajkiranBagal — APPROVED** [review]

**Comment by chandan-30** [comment]
> > @supriyajjadhav While there haven't been any modifications to the attributes.js file located in the deprecated folder, we decided to include it in order to maintain consistency.
> 
> We can remove it, if not needed

@supriyajjadhav I've removed it, Please review and approve.
CC: @RajkiranBagal

**Review by supriyajjadhav — APPROVED** [review]

**Pull request merged**